pkg/types/vsphere/platform.go

```diff
@@ -103,6 +107,11 @@ type Platform struct {
 	// FailureDomains is available in TechPreview.
 	// +kubebuilder:validation:Optional
 	FailureDomains []FailureDomain `json:"failureDomains,omitempty"`
+
+	// LoadBalancer defines how the load balancer used by the cluster is configured.
+	// LoadBalancer is available in TechPreview.
+	// +optional
+	LoadBalancer configv1.VSpherePlatformLoadBalancer `json:"loadBalancer,omitempty"`
```

Comment:
should this be using a pointer for `VSpherePlatformLoadBalancer` ?

Reply:
Yes, a pointer is preferred for an optional field.

Reply:
thanks